src/lib/pmd/PmdEngine.ts

```diff
@@ -34,7 +34,7 @@ export class PmdEngine implements RuleEngine {
 		return PmdEngine.NAME;
 	}
 
-	/* eslint-disable @typescript-eslint/no-unused-vars */
+	/* eslint-disable no-unused-vars, @typescript-eslint/no-unused-vars */
```

Comment:
Should this be `eslint-disable-next-line`?

Reply:
Probably. I'll make that change.